linters/check_columns.py

```diff
@@ -456,7 +465,16 @@ def check_profile():
                 print_options(result)
                 delete_command(result, "profile_columns.list")
 
+    return len(result)
+
 
 if __name__ == "__main__":
-    check_history()
-    check_profile()
+    result1 = check_history()
+    result2 = check_profile()
+
+    result = result1 or result2
+    if result == 0:
+        print("All checks passed.")
+    else:
+        print("Some checks failed.")
```

Comment:
I would personally write this as
```suggestion
    if result1 or result2:
        print("Some checks failed.")
    else:
        print("All checks passed.")
```
The reversing of the order is because this way I can guarantee not needing any brackets in the logical expression. (I can explain further on request.)